tests/Functional/Schema/OracleSchemaManagerTest.php

```diff
@@ -241,6 +243,20 @@ public function testListTableDateTypeColumns(): void
         self::assertInstanceOf(DateTimeTzType::class, $columns['col_datetimetz']->getType());
     }
 
+    public function testListTableFloatTypeColumns(): void
+    {
+        $table = new Table('tbl_float');
+        $table->addColumn('col_float', Types::FLOAT);
+        $table->addColumn('col_real_float', Types::REAL);
+
+        $this->dropAndCreateTable($table);
+
+        $columns = $this->schemaManager->listTableColumns('tbl_float');
+
+        self::assertInstanceOf(FloatType::class, $columns['col_float']->getType());
+        self::assertInstanceOf(RealFloatType::class, $columns['col_real_float']->getType());
+    }
```

Comment:
This test looks like it's valid on any platform, not just Oracle. Can we move it somewhere else?

Reply:
This is a functional test only for Oracle ci. As far as I can see, each platform has its own tests.

Reply:
> As far as I can see, each platform has its own tests.

Sure, to test behavior that is specific to a certain platform. My point is: This test contains nothing that is specific to Oracle. I don't see why we should only run it there.

Reply:
Hmm, are you suggesting moving this test to SchemaManagerFunctionalTestCase to make it universal for all?

Reply:
Done, one test failed, but it's not my fault (I think) :).

Reply:
I did a small code refactor. Now the common properties of float columns are tested for all platforms, and additionally for MySQL with unsigned.